templates/partials/footer.html.njk

```diff
@@ -33,6 +33,36 @@
       </li>
     </ul>
   </nav>
+  <div class="c-sponsor-footer c-sponsors--footer">
+    <div class="l-container">
+      {# @lukaszklis: this is just copy-pasted from the sponsor page. #}
+      <div class="c-sponsors">
```

Comment:
You can extract the entire `c-sponsors` from here to a separate partial (similar to what we have in ”speaker-gallery.html.njk” and include it here and on the sponsors page. This way we’re going to have only one place in which we manage the list of all sponsors.